music21/mei/base.py

```diff
@@ -1272,67 +1272,63 @@ def makeMetadata(documentRoot):
     work = documentRoot.find(f'.//{MEI_NS}work')
     if work is not None:
         # title, subtitle, and movement name
-        meta = metaSetTitle(work, meta)
-        # composer
-        meta = metaSetComposer(work, meta)
+        meta = metaSetTitles(work, meta)
+        # composers, etc
+        meta = metaSetContributors(work, meta)
```

Comment:
As far as I can tell the original function allowed multiple composer assignment as well. See the previous comment on `metaSetComposer` about `composer(s)`. The XPath query simply looks for all the `persName` elements within the responsibility statement.

Reply:
Yes, `metaSetComposer` was doing multiple composers.  The new version does multiple of _any_ contributor, using add(role) for any roles that are standard uniqueName keys (like 'composer', 'lyricist', etc) and add('otherContributor') for any non-standard roles (and the non-standard role is set in c.role).

Reply:
I would be happy to do the MEI parser changes as a separate PR after this one is merged, though.

Reply:
I have reverted the MEI parser improvements, but there are still some changes necessitated by the new metadata implementation, and one egregious bug fix where md.composer (instead of md.composers) was being set to a list.